main.tf

```diff
@@ -235,18 +236,18 @@ resource "aws_iam_role_policy_attachment" "attach_iam_policy_to_iam_role" {
 
 resource "aws_cloudwatch_log_group" "lambda_asg_log_group" {
   count             = var.asg_scheduler != null ? 1 : 0
-  name              = "/aws/lambda/ec2-asg-scheduler-${random_id.this[0].id}"
+  name              = "/aws/lambda_functions/ec2-asg-scheduler-${random_id.this[0].id}"
```

Comment:
the cloudwatch_log_group name is with the "/aws/lambda/" prefix: https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs.html

Reply:
🙏